Realm/RLMObjectSchema.mm

```diff
@@ -128,20 +128,6 @@ + (instancetype)schemaForObjectClass:(Class)objectClass {
     }]];
     schema.computedProperties = computedProperties;
 
-    // verify that we didn't add any properties twice due to inheritance
```

Comment:
I believe the original rationale was that the new ivar-based property detection made this check redundant. That being said, it doesn't make sense to remove the tests and I'm not convinced the check is actually redundant. I'll restore both.